radionets/evaluation/utils.py

```diff
@@ -267,10 +256,6 @@ def get_images(test_ds, num_images, norm_path="none", rand=False):
     if rand:
         indices = torch.randint(0, len(test_ds), size=(num_images,))
     img_test = test_ds[indices][0]
-    norm = "none"
-    if norm_path != "none":
-        norm = pd.read_csv(norm_path)
-    img_test = do_normalisation(img_test, norm)
     img_true = test_ds[indices][1]
     if len(img_true.shape) == 3:
```

Comment:
Do we still need that? I would have to squeeze again, when using a list with len(list.shape)==3

Reply:
Why does this have an effect on squeezing? This only applies a no longer supported normalisation?

Reply:
I was only talking about 260 and 261. My list is of shape [batch size, num. of component, attributes], thus the length is equal 3. And when opening an image with that function, I get an extra axis cause of this unsqueeze.

Reply:
Have to have another look into https://github.com/radionets-project/radionets/pull/116/files